jit/v2/JIT_v2.0.0.yaml

```diff
@@ -305,29 +305,29 @@ paths:
               examples:
                 badRequestExample:
                   summary: |
-                    Estimated Timestamp missing portCallServiceID
+                    Timestamp missing portCallServiceID
                   description: |
-                    `portCallServiceID` is a mandatory property in the `Estimated Timestamp`. This is an example of how the error object would look in case this property is missing
+                    `portCallServiceID` is a mandatory property in the `Timestamp`. This is an example of how the error object would look in case this property is missing
 
                     **NB**: `errorCode` not yet standardized by DCSA. Value `7003` is just a "random example"
                   value:
                     httpMethod: PUT
-                    requestUri: /v2/port-call-services/085a3207-5e45-49cf-8e1b-f8442beaf545/estimated-timestamp
+                    requestUri: /v2/port-call-services/085a3207-5e45-49cf-8e1b-f8442beaf545/timestamp
                     statusCode: 400
                     statusCodeText: Bad Request
                     statusCodeMessage: |
-                      portCallServiceID object not found - it is a mandatory property in Estimated Timestamp
+                      portCallServiceID object not found - it is a mandatory property in Timestamp
                     providerCorrelationReference: 4426d965-0dd8-4005-8c63-dc68b01c4962
                     errorDateTime: '2024-11-21T09:41:00Z'
                     errors:
                       - errorCode: 7003
                         property: portCallServiceID
                         errorCodeText: mandatory property missing
                         errorCodeMessage: |
-                          portCallServiceID must be provided as part of an Estimated Timestamp
+                          portCallServiceID must be provided as part of a Timestamp
         '404':
           description: |
-            In case the provider does not know of the `portCallServiceID` used in the request (this could be because of a `PUT` request that has not finished processing or simply because the resource does not exist) - it is possible for the provider to reject the requests by returning a `404` (Not Found).
+            In case the provider does not know of the `portCallServiceID` used in the request (this could be because of a `PUT` request, for creating/initiatin the **Port Call Service**, that has not finished processing or simply because the resource does not exist) - it is possible for the provider to reject the requests by returning a `404` (Not Found).
```

Comment:
The part between `(...)` could better be put in a separate line. 
E.g.:
In case the provider does not know of the `portCallServiceID` used in the request, it is possible for the provider to reject the requests by returning a `404` (Not Found). This could be because of a `PUT` request, for creating/initiating the **Port Call Service**, that has not finished processing or simply because the resource does not exist.

Reply:
I have updated the description (y)